### ISSUE TITLE
Extract NLog extensions to their own repository

### ISSUE THREAD

**Issue by snakefoot** [NLog/NLog#4952]
Could be nice if these extensions could be extracted to other repository:

- [x] NLog.Wcf
- [x] NLog.WindowsIdentity
- [x] NLog.PerformanceCounter
- [x] NLog.MSMQ
- [x] NLog.Database (Remember to create AOT-test-project)
- [x] NLog.WindowsRegistry
- [ ] NLog.OutputDebugString
- [x] NLog.Targets.Network
- [x] NLog.Targets.WebService
- [x] NLog.Targets.Mail
- [x] NLog.Targets.ConcurrentFile
- [x] NLog.Targets.Trace
- [ ] NLog.RegEx

It would reduce build-time / unit-test-time for the NLog-core-project. And can skipped in the normal NLog-release-cycle

**Comment by snakefoot** [comment]
Created: https://github.com/NLog/NLog.MSMQ

**Comment by snakefoot** [comment]
Created: https://github.com/NLog/NLog.WCF (Missing AppVeyor-build-pipeline)

**Comment by snakefoot** [comment]
Created: https://github.com/NLog/NLog.WindowsIdentity (Missing AppVeyor-build-pipeline)

**Comment by snakefoot** [comment]
Created https://github.com/NLog/NLog.PerformanceCounter (Missing AppVeyor-build-pipeline)

**Comment by snakefoot** [comment]
Have now created the following repositories:
- https://github.com/NLog/NLog.Targets.Database (Missing AppVeyor-build-pipeline)
- https://github.com/NLog/NLog.Targets.ConcurrentFile (Missing AppVeyor-build-pipeline)

Have tried to extract the Git-History from the NLog-repo:
1. Create the destination-repo in Github and clone locally
2. Execute the following command in the source-repo (using wildcard to match the files to be extracted)
- `git format-patch --root -- **/*Database*`
3. Copy the patch-files in the source-repo into destination-repo
4. Execute the following command in the destination-repo:
- `for /r %i in (*.patch) do git am %i`
5. If failing to apply a patch-file, then rerun the patch like this:
- `git am --reject`
6. Fix any issues found in `.rej`-files and after fixing, then stage the changes, and commit with this command:
- `git am --continue`

**Comment by snakefoot** [comment]
Use the following command for NLog.Targets.NetworkTarget:
- `git format-patch --root -- **/*Socket* **/*Network* **/*Log4J* **/*IpAddress* **/*SysLog* **/*Gelf* **/*Splunk* **/*NLogViewer* **/*Chainsaw*`

- https://github.com/NLog/NLog.Targets.Network

**Comment by snakefoot** [comment]
The following code have now been extracted into their own repos with #5985
 
- https://github.com/NLog/NLog.Targets.ConcurrentFile

- https://github.com/NLog/NLog.Targets.Database

- https://github.com/NLog/NLog.Targets.Mail

- https://github.com/NLog/NLog.Targets.WebService

- https://github.com/NLog/NLog.Targets.Network

- https://github.com/NLog/NLog.WindowsRegistry (Without Git history)

- https://github.com/NLog/NLog.Targets.Trace (Without Git history)

@304NotModified Could be nice to have AppVeyor-pipeline created for these new isolated projects :)

**Comment by 304NotModified** [comment]
@snakefoot our we could move to github actions 😅

(Another year is another CI 😜)

Github Copilot could help with the migration 

What are your thoughts about this @snakefoot?

Example by Copilot: https://github.com/CharsetDetector/UTF-unknown/pull/181

**Comment by 304NotModified** [comment]
@snakefoot ive changed your role to project creator in appveyor. So besides the (maybe) move to github actions, this could help for this task in the short term.

**Comment by snakefoot** [comment]
> I've changed your role to project creator in appveyor.

I have received the hint that you cannot help, and that I should do it myself :)

> So besides the (maybe) move to github actions, this could help for this task in the short term.

I hate messing with build-pipelines. Have no desire migrating to yet another build-system, but will just stick with AppVeyor until it completely breaks. But more than willing to give the reins to whoever wants to dabble with github-actions (or whatever shiny object they desire to play with). 

If you can make CoPilot migrate everything to github-actions and verify that everything works, then please be my guest and give the magic command to CoPilot.

**Comment by snakefoot** [comment]
Seems you have given someone else "creator" rights in appveyor. This is what I have for the "nlog"-account after signin with Github:

<img width="602" height="240" alt="Image" src="https://github.com/user-attachments/assets/4af8e5fb-a6fc-4ba5-ad9e-d634ba33f3da" />

Still "Collaborator" and no "New Project"-option to click

**Comment by 304NotModified** [comment]
@snakefoot for some reason it was not saved successfully I guess. I retried it and after refresh it looks good. could you please try again?

**Comment by snakefoot** [comment]
Still "Collaborator"-status and no "New Project"-option to click, when having selected the "nlog"-account.

**Comment by 304NotModified** [comment]
Yes it has been reverted again. I refreshed yesterday 10 times and it was OK, now it's indeed again "user"...

**Comment by 304NotModified** [comment]
I removed and added you, you have an invitation

**Comment by snakefoot** [comment]
Still User and Collaborator :(

**Comment by snakefoot** [comment]
Can see that you have tried to update my rights yesterday:

<img width="694" height="289" alt="Image" src="https://github.com/user-attachments/assets/e04cdec0-ca98-41ea-9055-e79d97697304" />

It seems `"isCollaborator": true,` is seperate from `"roleName": "Project creator",`

But the `"roleName": "Project creator",` do not allow me to click the "New Project"-option, when looking at the Project-list.

**Comment by 304NotModified** [comment]
@snakefoot do you login with your email or github account? 

I changed some other stuff, it should work now when you login with your github account 

Project creator is configured as follows:

![Image](https://github.com/user-attachments/assets/f1f52165-c505-4d40-b312-fd97d3c423f7)

**Comment by 304NotModified** [comment]
Ah the github team is wrong, as your also member of nlog/core-members

**Comment by snakefoot** [comment]
Guess you must invite me again, since I'm no longer member of NLog. But yes I'm doing signin using my Github-account.

**Comment by 304NotModified** [comment]
I don't understand why that is needed as I mapped the github teams to appveyor. But anyway, I've added you manually again. (Use the last request)

**Comment by snakefoot** [comment]
Looks like it works without clicking the invite. I now have the "New project"-option. Thank you for being persistent.

**Comment by snakefoot** [comment]
Was able to click the "New project"-option for all 8 projects, and all builds executed successfully.

**Comment by snakefoot** [comment]
Updated the NLog extension page with #6003
